lib/errorMap.ts

```diff
@@ -0,0 +1,16 @@
+import * as zod from 'zod';
+
+import { fromZodIssue } from './fromZodIssue';
+
+export const errorMap: zod.ZodErrorMap = (issue, ctx) => {
```

Comment:
[question] Should we also update our docs with the usage of `errorMap` as suggested [here](https://github.com/causaly/zod-validation-error/issues/36#issuecomment-1879874022)?

Reply:
We already cover `errorMap` in the docs, no?

Reply:
I was referring to the special use-case of  `zod-validation-error` with `react-hook-form` because the latter one is a popular library as well.